XLS-0033d-multi-purpose-tokens/README.md

```diff
@@ -365,7 +389,7 @@ Arbitrary metadata about this issuance, in hex format. The limit for this field
   "TransactionType": "MPTokenIssuanceCreate",
   "Account": "rajgkBmMxmz161r8bWYH7CQAFZP5bA9oSG",
   "AssetScale": "2", // <-- Divisible into 100 units / 10^2
-  "MaximumAmount": "5F5E100", //  <-- 100,000,000 (Hex)
+  "MaximumAmount": "100000000", //  <-- 100,000,000
```

Comment:
Change the language for this listing to `js` instead of `json`?
![image](https://github.com/user-attachments/assets/0919bb9c-3fb2-47f3-ab68-6fac37eb9d86)

Reply:
Good catch - updated via 53f4dfed3070e8cd275601d017bf84acbfa9ced8. The red highlight goes away in my preview; does it look right on your end?